allennlp/training/trainer.py

```diff
@@ -277,16 +277,28 @@ def _metrics_to_tensorboard(self,
                                 epoch: int,
                                 train_metrics: dict,
                                 val_metrics: dict = None) -> None:
+        """
+        send all of the train metrics (and validation metrics, if provided) to tensorboard
+        """
+        for name, value in train_metrics.items():
+            self._tensorboard.add_train_scalar(name, value, epoch)
+            if val_metrics:
+                self._tensorboard.add_validation_scalar(name, val_metrics[name], epoch)
+
+    def _metrics_to_console(self,                               # pylint: disable=no-self-use
```

Comment:
Spacing seems odd here; we've typically just been doing two spaces after the end of the line.